photos/main.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Spencer Kimball (spencer.kimball@gmail.com)
+
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/signal"
+	"reflect"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/cockroachdb/cockroach/util/log"
+	"github.com/cockroachdb/cockroach/util/stop"
+	"github.com/spf13/cobra"
+)
+
+// pflagValue wraps flag.Value and implements the extra methods of the
+// pflag.Value interface.
+type pflagValue struct {
+	flag.Value
+}
+
+func (v pflagValue) Type() string {
+	t := reflect.TypeOf(v.Value).Elem()
+	return t.Kind().String()
+}
+
+func (v pflagValue) IsBoolFlag() bool {
+	t := reflect.TypeOf(v.Value).Elem()
+	return t.Kind() == reflect.Bool
+}
+
+func normalizeStdFlagName(s string) string {
+	return strings.Replace(s, "_", "-", -1)
+}
+
+var usage = map[string]string{
+	"db":    "host:port of the CockroachDB cluster",
+	"certs": "directory of the client certificates",
+	"users": "number of concurrent simulated users",
+}
+
+// A Context holds configuration data.
+type Context struct {
+	// DBAddr is the <host:port> address of the database server.
+	DBAddr string
+	// CertsDir is the directory of the security certificates.
+	CertsDir string
+	// NumUsers is the number of concurrent users generating load.
+	NumUsers int
+	//
+	DB *sql.DB
+}
+
+var ctx = Context{
+	DBAddr:   "localhost:15432",
+	NumUsers: 1,
+}
+
+var loadCmd = &cobra.Command{
+	Use:   "photos",
+	Short: "generate artifical load using ",
+	Long: `
+Create artificial load using a simple database schema containing
+users, photos and comments. Users have photos, photos have comments.
+Users can author comments on any photos. User actions are simulated
+using an exponential distribution on user IDs, so lower IDs see
+more activity than high ones.
+`,
+	Example: `  photos --db=<host:port>`,
+	RunE:    runLoad,
+}
+
+func runLoad(c *cobra.Command, args []string) error {
+	log.Infof("generating load for %d concurrent users...", ctx.NumUsers)
+	db, err := openDB(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer func() { _ = db.Close() }()
+	if err := initSchema(db); err != nil {
+		log.Fatal(err)
+	}
+
+	stopper := stop.NewStopper()
```

Comment:
So that users who see our examples don't copy-and-paste a pattern that will change/go-away. Similarly, you should probably be using the stdlib `log` package instead of the `cockroach/util/log` package. The latter seems more important than getting rid of the use of `util/stop`.